openeo/basics.ipynb

```diff
@@ -12,132 +12,522 @@
+       " 'SENTINEL2_L1C',\n",
+       " 'SENTINEL2_L2A',\n",
+       " 'SENTINEL1_GRD',\n",
+       " 'COPERNICUS_30']"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "print(connection.list_collection_ids())"
+    "#check for available list of collection\n",
+    "connection.list_collection_ids()"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "ece57c0c-2965-4d13-b0e2-2661d7f1d40d",
+   "id": "9ea2aab6",
    "metadata": {},
    "source": [
     "**Print 1st collection from the list**"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "085ee4f5-bc20-4a21-b7c7-fe0a67d1126b",
+   "execution_count": 5,
+   "id": "e99fa9cc",
    "metadata": {
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'description': 'The Plant Phenology Index (PPI) is a physically based vegetation index, that is optimized for monitoring vegetation phenology and has a linear relationship with green leaf area index, a strong correlation with gross primary productivity, and is shown capable of disentangling remotely sensed plant phenology from snow seasonality. The PPI Seasonal Trajectories product is derived from a function fitting of the time series of the raw PPI values and hence provides a filtered time series of PPI, with regular time steps, and related quality information. The product is derived from Copernicus Sentinel-2 satellite observations and updated annually.',\n",
+       " 'extent': {'spatial': {'bbox': [[-25, 27, 45, 72]]},\n",
+       "  'temporal': {'interval': [['2017-01-01T00:00:00Z',\n",
+       "     '2022-12-31T00:00:00Z']]}},\n",
+       " 'id': 'SEASONAL_TRAJECTORIES',\n",
+       " 'keywords': ['PPI',\n",
+       "  'index',\n",
+       "  'land',\n",
+       "  'vegetation',\n",
+       "  'plant',\n",
+       "  'phenology',\n",
+       "  'growing season',\n",
+       "  'productivity',\n",
+       "  'sentinel'],\n",
+       " 'license': 'proprietary',\n",
+       " 'links': [{'href': 'https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-vegetation-phenology-and-productivity',\n",
+       "   'rel': 'alternate'}],\n",
+       " 'stac_extensions': ['https://stac-extensions.github.io/datacube/v2.2.0/schema.json',\n",
+       "  'https://stac-extensions.github.io/eo/v1.1.0/schema.json'],\n",
+       " 'stac_version': '0.9.0',\n",
+       " 'title': 'Sentinel-2 derived vegetation indices (NDVI, FAPAR, PPI and LAI) at 10m resolution over European mainland. These indices are provided by the High Resolution Vegetation Phenology and Productivity service, part of the Copernicus land monitoring service.'}"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "print(connection.list_collections()[1])"
+    "connection.list_collections()[1]"
```

Comment:
I don't think using `connection.list_collections()[1]`  is something to suggest in a "basics" notebook, it's not very practical and the dict dump is very ugly/unreadable.

Reply:
updated